pkg/time/time.go

```diff
@@ -8,3 +8,15 @@ import (
 // variable behavior, tracking of calls, etc depending on what the test
 // needs.
 var After = time.After
+
+// Time is a function that helps convert test time values
+// to pointers for the json STATUS
+func Time(time time.Time) *time.Time {
+	return &time
+}
+
+// Duration is a function that helps convert test duration values
+// to pointers for the json STATUS
+func Duration(duration time.Duration) *time.Duration {
+	return &duration
+}
```

Comment:
I'd remove these; either move them into the test if you really want those helpers but it looks like you just need 2 test values for reuse:
```
testTime := time.Date(...)
testDuration := time.Duration(100) * time.Second
...
...
StartTime:=&testTime
Duration := &testDuration
...
```

If something is mutating those and you need new values each time, then I'd just move these helpers.